spec/controllers/api_v1_slides_controller_spec.rb

```diff
@@ -24,6 +34,10 @@
       get :show, params: { id: slide.id }, format: 'json'
       expect(response.status).to eq(200)
       expect(assigns(:slide)).to eq(slide)
+      json = JSON.parse(response.body);
+      list_json_keys.each do |k|
+        expect(json.has_key?(k)).to eq(true)
```

Comment:
__[rubocop]__
* Style/PreferredHashMethods: Use `Hash#key?` instead of `Hash#has_key?`. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-key)

<!-- Comment by [SideCI](https://sideci.com) -->